scripts/exglobal_cleanup.sh

```diff
@@ -0,0 +1,214 @@
+#! /usr/bin/env bash
+
+source "${HOMEgfs}/ush/preamble.sh"
+
+###############################################################
+# Clean up previous cycles; various depths
+# PRIOR CYCLE: Leave the prior cycle alone
+# shellcheck disable=SC2153
+GDATE=$(${NDATE} -"${assim_freq}" "${PDY}${cyc}")
+
+# PREVIOUS to the PRIOR CYCLE
+GDATE=$(${NDATE} -"${assim_freq}" "${GDATE}")
+gPDY="${GDATE:0:8}"
+gcyc="${GDATE:8:2}"
+
+# Remove the TMPDIR directory
+# TODO Only prepbufr is currently using this directory, and all jobs should be
+#   cleaning up after themselves anyway
+COMIN="${DATAROOT}/${GDATE}"
+[[ -d ${COMIN} ]] && rm -rf "${COMIN}"
+
+if [[ "${DELETE_COM_IN_ARCHIVE_JOB:-YES}" == NO ]] ; then
+    exit 0
+fi
+
+# Step back every assim_freq hours and remove old rotating directories
+# for successful cycles (defaults from 24h to 120h).
+# Retain files needed by Fit2Obs
+# TODO: This whole section needs to be revamped to remove marine component
+#  directories and not look at the rocoto log.
+GDATEEND=$(${NDATE} -"${RMOLDEND:-24}"  "${PDY}${cyc}")
+GDATE=$(${NDATE} -"${RMOLDSTD:-120}" "${PDY}${cyc}")
+RTOFS_DATE=$(${NDATE} -48 "${PDY}${cyc}")
```

Comment:
Perhaps make `RTOFS_DATE` look-back configurable as well? `RMOLDRTOFS` or something similar and make 48hrs the default?

Reply:
Added